src/Makefile.am.inc

```diff
@@ -40,6 +40,7 @@ eos_metrics_instrumentation_SOURCES = \
 eos_metrics_instrumentation_CPPFLAGS = \
 	$(EOS_INSTRUMENTATION_CFLAGS) \
 	-DINSTRUMENTATION_CACHE_DIR="\"$(instrumentationcachedir)\"" \
+	-DSYS_CONF_DIR="\"$(sysconfdir)\"" \
```

Comment:
Nitpick: `-DSYSCONFDIR` would be a bit more greppable (`git grep -i sysconfdir`).